backend/models/index.js

```diff
@@ -18,6 +20,19 @@ const sequelize = new Sequelize(
   }
 );
 
+const sequelizePostgres = new Sequelize(
+  postgresConfig.database,
+  postgresConfig.username,
+  postgresConfig.password,
+  {
+    host: postgresConfig.host,
+    dialect: postgresConfig.dialect,
+    dialectOptions: {
+      ssl: true,
+    },
```

Comment:
Is there a way to enforce read-only mode on connection? We should not let people to alter our databases via WAMP calls, you know :smile:

Reply:
Just for the track, we decided to enforce that with postgres read-only access user